src/contribute/design-documents.md

```diff
@@ -0,0 +1,87 @@
+- Configurable BeamFileSystem [[doc](https://docs.google.com/document/d/1-7vo9nLRsEEzDGnb562PuL4q9mUiq_ZVpCAiyyJw8p8)]
+- New API for writing files in Beam [[doc](http://s.apache.org/fileio-write)]
+
+### Metrics
+- Metrics API [[doc](http://s.apache.org/beam-metrics-api)]
+- I/O Metrics [[doc](https://s.apache.org/standard-io-metrics)]
+- Metrics extraction independent from runners / execution engines [[doc](https://s.apache.org/runner_independent_metrics_extraction)]
+
+### Runners
+- Runner Authoring Guide [[doc](https://s.apache.org/beam-runner-guide)] (obsoletes [[doc](http://s.apache.org/beam-runner-api)] and [[doc](https://s.apache.org/beam-runner-1-pager)])
+- Composite PInputs, POutputs, and the Runner API [[doc](https://s.apache.org/beam-runner-composites)]
+- Side Input Architecture for Apache Beam [[doc](https://s.apache.org/beam-side-inputs-1-pager)]
+
+### SQL / Schema
+- Streams and Tables [[doc](https://s.apache.org/beam-streams-tables)]
+- Streaming SQL [[doc](http://s.apache.org/streaming-sql-spec)]
+- Schema-Aware PCollections [[doc](https://docs.google.com/document/d/1tnG2DPHZYbsomvihIpXruUmQ12pHGK0QIvXS1FOTgRc)]
+
+### Portability
+- Fn API
+  - Apache Beam Fn API Overview [[doc](https://s.apache.org/beam-fn-api)]
+  - Processing a Bundle [[doc](https://s.apache.org/beam-fn-api-processing-a-bundle)]
+  - Progress [[doc](https://s.apache.org/beam-fn-api-progress-reporting)]
+  - Graphical view of progress [[doc](https://docs.google.com/document/d/1Dx18qBTvFWNqwLeecemOpKfleKzFyeV3Qwh71SHATvY)]
+  - Fn State API and Bundle Processing [[doc](https://s.apache.org/beam-fn-state-api-and-bundle-processing)]
+  - How to send and receive data [[doc](https://s.apache.org/beam-fn-api-send-and-receive-data)]
+  - Defining and adding SDK Metrics [[doc](https://s.apache.org/beam-fn-api-metrics)]
+  - SDK harness container contract [[doc](https://s.apache.org/beam-fn-api-container-contract)]
+  - Structure and Lifting of Combines [[doc](https://s.apache.org/beam-runner-api-combine-model)]
+- Cross-language Beam Pipelines [[doc](https://s.apache.org/beam-mixed-language-pipelines)]
+- SDK X with Runner Y using Runner API [[doc](https://s.apache.org/beam-job-api)]
+- Flink Portable Runner Overview [[doc](https://s.apache.org/portable-flink-runner-overview)]
+
+### Testing
+- More Expressive PAsserts [[doc](https://docs.google.com/document/d/1fZUUbG2LxBtqCVabQshldXIhkMcXepsbv2vuuny8Ix4)]
+- Mergebot design document [[doc](https://docs.google.com/document/d/18iFnW6egjqd_ADXCTQcuAkkz3J96LHdV5DlYUhXHf0M)]
+- Performance tests for commonly used file-based I/O PTransforms [[doc](https://docs.google.com/document/d/1dA-5s6OHiP_cz-NRAbwapoKF5MEC1wKps4A5tFbIPKE)]
+- Performance tests results analysis and basic regression detection [[doc](https://docs.google.com/document/d/1Cb7XVmqe__nA_WCrriAifL-3WCzbZzV4Am5W_SkQLeA)]
+
+### Python
+- Beam Python User State and Timer APIs [[doc](https://s.apache.org/beam-python-user-state-and-timers)]
+
```

Comment:
There are no other documents for the Python SDK? @robertwb

Reply:
Python Kafka connector: https://docs.google.com/document/d/1ogRS-e-HYYTHsXi_l2zDUUOnvfzEbub3BFkPrYIOawU/edit

Reply:
Done